javaagent-tooling/src/main/java/io/opentelemetry/javaagent/tooling/muzzle/matcher/MuzzleGradlePluginUtil.java

```diff
@@ -135,55 +120,37 @@ public static void assertInstrumentationMuzzled(
   }
 
   /**
-   * Prints all references from all instrumenters present in the passed {@code
+   * Prints all references from all instrumentation modules present in the passed {@code
    * instrumentationClassLoader}.
    *
    * <p>Called by the {@code printMuzzleReferences} gradle task.
    */
   public static void printMuzzleReferences(ClassLoader instrumentationClassLoader) {
-    for (Object instrumenter : loadAllInstrumenters(instrumentationClassLoader)) {
-      if (instrumenter instanceof Instrumenter.Default
-          || instrumenter instanceof InstrumentationModule) {
+    for (InstrumentationModule instrumentationModule :
+        ServiceLoader.load(InstrumentationModule.class, instrumentationClassLoader)) {
+      try {
+        Method getMuzzleMethod =
+            instrumentationModule.getClass().getDeclaredMethod("getMuzzleReferenceMatcher");
+        ReferenceMatcher muzzle;
         try {
-          Method getMuzzleMethod =
-              instrumenter.getClass().getDeclaredMethod("getMuzzleReferenceMatcher");
-          ReferenceMatcher muzzle;
-          try {
-            getMuzzleMethod.setAccessible(true);
-            muzzle = (ReferenceMatcher) getMuzzleMethod.invoke(instrumenter);
-          } finally {
-            getMuzzleMethod.setAccessible(false);
-          }
-          System.out.println(instrumenter.getClass().getName());
-          for (Reference ref : muzzle.getReferences()) {
-            System.out.println(prettyPrint("  ", ref));
-          }
-        } catch (Exception e) {
-          System.out.println(
-              "Unexpected exception printing references for " + instrumenter.getClass().getName());
-          throw new RuntimeException(e);
+          getMuzzleMethod.setAccessible(true);
+          muzzle = (ReferenceMatcher) getMuzzleMethod.invoke(instrumentationModule);
+        } finally {
+          getMuzzleMethod.setAccessible(false);
+        }
+        System.out.println(instrumentationModule.getClass().getName());
+        for (Reference ref : muzzle.getReferences()) {
+          System.out.println(prettyPrint("  ", ref));
         }
```

Comment:
just checking if intentional

Reply:
Sysout? Yes, this class is just a helper that's only run from the Gradle muzzle plugin.

Reply:
oh sorry! i missed that this is inside of `printMuzzleReferences` 😁